### PR TITLE
Update capabilities status

### PR DESCRIPTION
**Pull request by beaufortfrancois**
This PR makes sure the "New capabilities status" page matches latest articles status:
- Move "Content Indexing API" from OT to Stable
- Move "Notification Triggers" from behind a flag to OT
- Order alphabetically capabilities in tables
- Remove Badging API outdated note
- Update "Content Indexing API" article to reflect shipped status

@petele @tomayac Can you review?

Live preview:
- https://deploy-preview-4503--web-dev-staging.netlify.app/fugu-status/
- https://deploy-preview-4503--web-dev-staging.netlify.app/content-indexing-api/

### CODE REVIEW

File: src/site/content/en/blog/fugu-status/index.md [hunk 1]
```diff
@@ -267,6 +257,21 @@ latest version of Chrome, and in many cases other Chromium based browsers.
           <em>Updated April 14, 2020</em>
         </td>
       </tr>
+      <tr>
+        <td>
+          <a href="/wakelock/">Wake Lock API</a>
```
Comment (tomayac) [inline src/site/content/en/blog/fugu-status/index.md:262]:
```suggestion
          <a href="/wake-lock/">Wake Lock API</a>
```

**Review by tomayac — APPROVED** [review]
LGTM, approval 

(Just fixed one URL, saving a round trip)

File: src/site/content/en/blog/fugu-status/index.md (hunk 1 above)
Comment (beaufortfrancois) [reply src/site/content/en/blog/fugu-status/index.md:262]: Done.

File: src/site/content/en/blog/fugu-status/index.md [hunk 2]
```diff
@@ -120,6 +92,20 @@ trials in the [Origin Trials Guide for Web Developers][ot-guide].
...
+      <tr>
+        <td>
+          <a href="/notification-triggers/">
+            Notification Triggers
+          </a>
+        </td>
+        <td>
+          Notification Triggers let you schedule notifications in advance, so
+          that the operating system will deliver the notification at the right
+          time - even if there is no network connectivity, or the device is in
```
Comment (mihajlija) [inline src/site/content/en/blog/fugu-status/index.md:104]:
```suggestion
          time-even if there is no network connectivity, or the device is in
```

File: src/site/content/en/blog/fugu-status/index.md [hunk 3]
```diff
@@ -227,6 +204,19 @@ latest version of Chrome, and in many cases other Chromium based browsers.
...
       </tr>
+      <tr>
+        <td>
+          <a href="/get-installed-related-apps/">
+            Get Installed Related Apps API
+          </a>
+        </td>
+        <td>
+          The Get Installed Related Apps API is a web platform API
+          that allows your web app to check to see if your iOS/Android app is
```
Comment (mihajlija) [inline src/site/content/en/blog/fugu-status/index.md:215]:
```suggestion
          that allows your web app to check to see if your iOS or Android app is
```

**Review by mihajlija — APPROVED** [review]

**Pull request merged**